desktop/webpack.config.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External Dependencies
+ */
+const path = require( 'path' );
+const { DefinePlugin } = require( 'webpack' );
+const CopyPlugin = require( 'copy-webpack-plugin' );
+
+const DEVELOPMENT = process.env.NODE_ENV === 'development';
+
+module.exports = {
+	context: path.resolve( __dirname, 'app' ),
+	output: {
+		path: path.join( __dirname, 'dist' ),
+		filename: 'index.js',
+		libraryTarget: 'commonjs2',
+	},
+	plugins: [
+		new CopyPlugin( {
+			patterns: [
+				{
+					from: path.join( __dirname, 'public_desktop' ),
+					to: path.join( 'public_desktop' ),
+				},
+			],
+		} ),
+		new DefinePlugin( {
+			__public: `"${ path.join( __dirname, 'public_desktop' ).replace( /\\/g, '\\\\' ) }"`,
```

Comment:
Who is using this?
Seems duplicated with `desktop/app/lib/assets/index.js`?

Reply:
@nsakaimbo Was there a reason for the __public? I removed it and everything was still working.

Reply:
@belcherj `preload.js` is contained in `public_desktop` - this is a very important file and is required for IPC to work.  Other files such as icon assets, `preferences.html` etc are located here as well. Feel free to move it elsewhere, but be very careful to update references to this file everywhere in the app code.

Reply:
I don't follow... `new DefinePlugin( { __public: ... })` is defining a global variable named `__public`. But I can't find where in our code we access that variable.

Is that an electron thing?

Reply:
> I don't follow... new DefinePlugin( { __public: ... }) is defining a global variable named __public. But I can't find where in our code we access that variable.

Yeah that's a good catch. It's been a while since I worked on this bit so it's quite possible that that this is the approach I went with initially and dropped it entirely in favor of the logic in `desktop/app/lib/assets/index.js`. Removing/changing it is fine, my only suggestion would be to make sure that the following work in both the "development" app and built app: (1) IPC, (2), "About" window, (3) Preferences window. cc: @belcherj